templates/contacts.html.twig

```diff
@@ -65,22 +77,20 @@
                         <ul class="list-group list-group-flush">
                             <li class="list-group-item">
                                 <div class="row no-gutters">
-                                <div class="col-auto">
-                                    <img class="img-fluid event-img my-2 mr-2" src="{{ asset( "images/Romashka.jpg") }}" alt="rolendar">
+                                    <div class="col-auto">
+                                        <img class="img-fluid event-img my-2 mr-2" src="{{ asset( "images/Romashka.jpg") }}" alt="rolendar">
+                                    </div>
+                                    <div class="col">
+                                        <p class="card-text m-2">
+                                        <h5 > Соучредитель проекта <span class="bordered-text"> Роман Гаврилов </span></h5>
```

Comment:
@daria-sieroshtan 
Com'oooooon
Border round name loos like necrologue
You can use some combination of bold, italics and underline to highlight text

Reply:
It is meant to be so. This tribute mention.